lms/templates/theme-variables.html

```diff
@@ -357,7 +357,7 @@
     'cert_logo': get_value('certificates', {}).get('cert_logo', get_value('logo_positive', static('images/branding/brand-logo.svg'))),
     'logo_width': get_value('certificates', {}).get('cert_logo_width', '160px'),
     'platform_name': translate(get_value('certificates', {}).get('platform_name', get_value('PLATFORM_NAME', '')), default=_('Platform Name')),
-    'certificate_banner_text': get_value('certificates', {}).get('certificate_banner_text', course_context.get('document_banner', '')),
+    'certificate_banner_text': translate(get_value('certificates', {}).get('certificate_banner_text', course_context.get('document_banner', ''))),
```

Comment:
what's the purpose of `translate` here? I don't think it's relevant. right?

Reply:
@OmarIthawi the `translate` is coming from a commit I still need to push from a rebase.  All the other values are `translate`d, so I was following the pattern there.  It probably wouldn't be relevant unless the value is not set and we fall back to the default, in which case we may have a translation string in place.

I don't have permission to force-push to this repo though.

Reply:
Hmm, wait,  I was confused. I forgot, my rebase was only for squashing. 😆 Still, point stands about the translate.
I'll merge as is tomorrow unless @OmarIthawi or @grozdanowski one of you might be able to give me force-push perms on this repo.